test_runner/fixtures/benchmark_fixture.py

```diff
@@ -409,6 +410,42 @@ def record_pageserver_writes(
             report=MetricReport.LOWER_IS_BETTER,
         )
 
+    def record_pagebench_results(self, name: str, results: Dict[str, Any], duration: str):
+        total = results["total"]
+
+        metric = "request_count"
+        self.record(
+            metric_name=f"{name}.{metric}",
+            metric_value=total[metric],
+            unit="",
+            report=MetricReport.HIGHER_IS_BETTER,
+        )
+
+        metric = "latency_mean"
+        self.record(
+            metric_name=f"{name}.{metric}",
+            metric_value=humantime_to_ms(total[metric]),
+            unit="ms",
+            report=MetricReport.LOWER_IS_BETTER,
+        )
+
+        metric = "latency_percentiles"
+        for k, v in total[metric].items():
+            self.record(
+                metric_name=f"{name}.{metric}.{k}",
+                metric_value=humantime_to_ms(v),
+                unit="ms",
+                report=MetricReport.LOWER_IS_BETTER,
+            )
+
+        metric = "duration"
+        self.record(
+            metric_name=f"{name}.{metric}",
+            metric_value=humantime_to_ms(duration) / 1000,
+            unit="s",
+            report=MetricReport.LOWER_IS_BETTER,
```

Comment:
If we can run `pagebench` only with `--runtime` argument, then we can mark it as a parameter value:

```suggestion
            report=MetricReport.TEST_PARAM,
```